.gitignore

```diff
@@ -2,3 +2,4 @@
 .Rhistory
 .RData
 .Ruserdata
+*/out/*
```

Comment:
Also, one thing to mention is a tip with the "do not commit" to files in out folder. So it is good to have an .empty file inside of the folder (before ignoring all outputs) where things do not get committed to but still holds the structure of the repo.